lib/on_form/form.rb

```diff
@@ -91,7 +95,13 @@ def self.expose_collection_of(association_name, on: nil, prefix: nil, suffix: ni
       collection_form_class.take_identity_from singular_name, convert_to_model: false
       collection_form_class.class_eval(&block)
 
-      define_method(exposed_name) { collection_wrappers[association_name] ||= CollectionWrapper.new(backing_model_instance(on), association_name, collection_form_class, allow_insert, allow_update, allow_destroy) } # used by action_view's fields_for, and by the following lines
+      define_method(exposed_name) do
+        collection_wrappers[association_name] ||= CollectionWrapper.new(
+          backing_model_instance(on), association_name, collection_form_class,
+          options.slice(:allow_insert, :allow_update, :allow_destroy, :reject_if)
+        )
+      end
+      # used by action_view's fields_for, and by the following lines
```

Comment:
Can this be moved up above the method definition? Not clear to me what it refers to at the moment